tests/test_executable.py

```diff
@@ -121,6 +121,25 @@ def test_check_args_stdio_valid(self):
         args = ['report', '--reporter', 'stdio']
         self.check_args_tester(args, expected_fail=False)
 
+    def test_merge_option(self):
+        """Test test_merge_option() with some arguments."""
```

Comment:
The docstring should say what is being tested, not the name of the test, so in this case it should say something like `Test parsing of multiple different merge options`